Specs/Sequence/Chinese/QuotedTextModel.json

```diff
@@ -0,0 +1,28 @@
+[
+	{
+		"Input": " 这个“数字”是『123』或者『456』",
+		"Results": [
```

Comment:
As this new type only exists in .NET, you should add one annotation per spec case to disable tests in the other platforms.
"NotSupportedByDesign": "javascript, python, java",

Reply:
I find that sometime when NotSupportedByDesign is added in the QuotedTextModel json file, the tests will fail.

Reply:
This should not happen. I'll take a look. Sorry for the delay.

Reply:
Thanks 👍